### DISCUSSION TITLE
cannot perform conditional operations in nested select.

### DISCUSSION THREAD

**Discussion by rano667** [prisma/prisma#20079] [Q&A]
## Problem

```js
 const feeds = await prisma.feeds.findMany({
       select: {
         CreatedTime: true,
         S3Key: true,
         UsersFeeds: {
           select: {
             UserDetails: {
               select: {
                 Name: true,
                 ProfilePicture: true,
                 CompanyUsers: {
                   select: {
                     CompanyDetails: {
                       select: {
                         CompanyName: true
                       }
                     }
                   }
                 }
               },
               where: {
               ~~~~~
                 isNot: {
                   UD_ID: 'b0beafa6-50f6-ed11-a528-7c5079a473a0'
                 }
               }
             }
           }
         }
       },
       orderBy: {
         CreatedTime: 'desc'
       },
       skip: 0,
       take: 20
     })
```
```js
Unknown arg `where` in select.UsersFeeds.select.UserDetails.where for type UserDetails. 
Did you mean `select`? Available args:
type UserDetails {

}
```
## Suggested solution

why the where clause is not working??
Please provide support.

**Comment by nurul3101** [comment]
Hello @rano667 👋 

Can you please share your schema file?
It would help us in reproducing this issue.

**Reply by rano667** [reply]
sorry cant share property of company!
cant u create example schema and try it.
It's working great without the where clause, I would like to say.

**Reply by nurul3101** [reply]
It looks like you're trying to use a where condition within the select statement, which is not recognised. I would recommend you to move the where clause at the top-level instead of nesting it in select.
Here's an [example](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#notin) in which where clause is at the top level.

**Reply by rano667** [reply]
Tried moving the where clause at the top-level, but doesn't help

**Reply by nurul3101** [reply]
What error do you get? The same error?

**Reply by nurul3101** [reply]
Please feel free to let us know in case you still need assistance.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏

**Reply by rano667** [reply]
Thanks for the support and time. We have moved on from that problem.
